snuba/web/rpc/v1/resolvers/R_uptime_checks/resolver_time_series.py

```diff
@@ -0,0 +1,276 @@
+import uuid
+from collections import defaultdict
+from datetime import datetime
+from typing import Any, Dict, Iterable
+
+from google.protobuf.json_format import MessageToDict
+from google.protobuf.timestamp_pb2 import Timestamp
+from sentry_protos.snuba.v1.endpoint_time_series_pb2 import (
+    DataPoint,
+    TimeSeries,
+    TimeSeriesRequest,
+    TimeSeriesResponse,
+)
+from sentry_protos.snuba.v1.request_common_pb2 import TraceItemType
+
+from snuba.attribution.appid import AppID
+from snuba.attribution.attribution_info import AttributionInfo
+from snuba.datasets.entities.entity_key import EntityKey
+from snuba.datasets.entities.factory import get_entity
+from snuba.datasets.pluggable_dataset import PluggableDataset
+from snuba.query import OrderBy, OrderByDirection, SelectedExpression
+from snuba.query.data_source.simple import Entity
+from snuba.query.dsl import Functions as f
+from snuba.query.dsl import column
+from snuba.query.logical import Query
+from snuba.query.query_settings import HTTPQuerySettings
+from snuba.request import Request as SnubaRequest
+from snuba.web.query import run_query
+from snuba.web.rpc.common.common import (
+    attribute_key_to_expression,
+    base_conditions_and,
+    trace_item_filters_to_expression,
+    treeify_or_and_conditions,
+)
+from snuba.web.rpc.common.debug_info import (
+    extract_response_meta,
+    setup_trace_query_settings,
+)
+from snuba.web.rpc.v1.resolvers import ResolverTimeSeries
+from snuba.web.rpc.v1.resolvers.R_uptime_checks.common.aggregation import (
+    aggregation_to_expression,
+    get_count_column,
+)
+
+
+def _convert_result_timeseries(
```

Comment:
I think this function we could definitely move to the higher level common module